makefile

```diff
@@ -2,24 +2,24 @@
 
 #Variables
 APP_NAME=ps-tag-onboarding-go
-COMPOSE_FILE=.docker/stack.yaml
+COMPOSE_FILE=build/.docker/docker-compose.yaml
 
 # Tasks
 default: run-with-docs
```

Comment:
A couple things that should be updated in the makefile
1. By default make does not force a build. I would expect it to. If you wanted to run with out building you could directly run docker-compose or a specific makefile target.
2. If make starts docker-compose there should be a target to stop the application with out doing a full clean.